cmds/modules/noded/main.go

```diff
@@ -193,6 +195,27 @@ func action(cli *cli.Context) error {
 		return hypervisor, nil
 	})
 
+	log.Info().Msg("start perf scheduler")
+	perfMon, err := perf.NewPerformanceMonitor("unix://var/run/redis.sock")
```

Comment:
Shouldn't be hard coded, specially that we already have a command line argument for that